lib/admin/route/admin.instance.create.id/admin.instance.create.id.controller.js

```diff
@@ -95,20 +92,17 @@ instanceCreateIdController.setData = (pageInstance, pageData) => {
         })
       if (existingIds.length) {
         pageData.setUserDataProperty('existingIds', true)
-        pageInstance = produce(pageInstance, draft => {
-          const existingItems = existingIds.map((_id, index) => {
-            const item = {
-              _id: `admin.instance.create.id--existing--radios--${index}`,
-              _type: 'radio',
-              value: _id,
-              label: getInstanceTitle(_id)
-            }
-            return item
-          })
-          const existingRadios = jp.query(draft, '$..[?(@._id == "admin.instance.create.id--existing--radios")]')[0]
-          existingRadios.items = existingItems
-          return draft
+        const existingItems = existingIds.map((_id, index) => {
+          const item = {
+            _id: `admin.instance.create.id--existing--radios--${index}`,
+            _type: 'radio',
+            value: _id,
+            label: getInstanceTitle(_id)
```

Comment:
The three uses of `_id` here, is the use on L97 meant to be a string? L99 and L100 look like values. Is this not going to blow up? Or are they all values?

Reply:
L97 - setting the _id of the radio option component (every component has an _id - even though this one is being dynamically generated)
L99 - setting the value of the radio option to be the _id of the component/page we're looping over
L110 - set the label to be the title value of the component/page we're looping over